oak_functions/loader/src/server.rs

```diff
@@ -559,9 +603,15 @@ impl WasmHandler {
         })
     }
 
-    fn init_wasm_state(&self, request_bytes: Vec<u8>) -> anyhow::Result<WasmState> {
+    fn init(&self, request_bytes: Vec<u8>) -> anyhow::Result<WasmState> {
         let mut extensions_indices = HashMap::new();
         let mut extensions_metadata = HashMap::new();
+        let mut extensions_endpoints = HashMap::new();
+
+        let mut channel_switchboard = ChannelSwitchboard::new();
+        // Register only lookup channel for now
+        let lookup_endpoint = channel_switchboard.register(ChannelHandle::LookupData);
+        extensions_endpoints.insert(LOOKUP_ABI_FUNCTION_NAME.to_string(), lookup_endpoint);
```

Comment:
So you are using the pre-existing function name to index in the map? That's fine, but it seems we need to be quite careful, since it was not meant for this (at least, I find it a bit unexpected).

Reply:
Yes, I added a comment. It is a workaround until I can integrate channels better into extensions. I have added a comment on `let mut extension_endpoints`.